src/components/App/App.js

```diff
@@ -61,15 +87,21 @@ function App() {
     return (
       <main>
         <aside>
-          <p>Welcome, user id: { currentUser.uid }</p>
+          <img className="sprite-logo" src={sprite_logo}/>
+          <p>{ currentUser.uid }</p>
           <p>Verified: {currentUser.emailVerified ? 'Yes' : 'Not Yet'}</p>
-          <button type="button" onClick={formSignOff}>Sign Out</button>
+          <div className="audio-control">
+            <img src={mute_icon} />
+            <img src={deafen_icon} />
+          </div>
```

Comment:
Next sprint you'll probably make this interactive.

All good for me, approving!